python/integration_tests/test_task_worker_processing.py

```diff
@@ -32,7 +31,7 @@ def manage_taskworker(
     shutdown_event: threading.Event,
 ) -> None:
     print(f"[taskworker_{worker_id}] Starting taskworker_{worker_id}")
-    worker = SimpleTaskWorker(
+    worker = ConfigurableTaskWorker(
```

Comment:
Just FYI: I also updated this test in my PR. If this goes out first, I'll fix it: https://github.com/getsentry/taskbroker/pull/112